src/style.css

```diff
@@ -1,3 +1,83 @@
-body {
+.body {
```

Comment:
Change '.body' selector to 'body' for applying styles to the whole body of the page.